### DISCUSSION TITLE
Does it make any difference adding semicolon at the beginning of an alias

### DISCUSSION THREAD

**Discussion by yadavan88** [sbt/sbt#7259] [Q&A] [answered]
Hi,
I have always created the sbt alias in this format:
```
addCommandAlias("ci", ";clean;compile;test")
```
However, I am unclear what is the purpose of the semicolon at the beginning (;clean). Does it make any difference in some particular scenarios, or it can always be ignored? 

Thanks in advance

**Comment by eed3si9n** [comment, marked as answer]
Semilon `;` is a command in sbt to concatenate other commands, and `;clean;compile;test` used to be the only syntax. In sbt 1.3.0 (https://github.com/sbt/sbt/pull/4456) multi command was implemented so sbt engine itself is sort of aware of the semicolon `;`, and supports `clean;compile;test`.

> Does it make any difference in some particular scenarios, or it can always be ignored?

So starting sbt 1.3.0, you can drop the initial semicolon.
